core/src/main/java/io/substrait/expression/proto/ProtoExpressionConverter.java

```diff
@@ -318,4 +361,28 @@ public Expression.Literal from(io.substrait.proto.Expression.Literal literal) {
           "Unexpected value: " + literal.getLiteralTypeCase());
     };
   }
+
+  private static List<FunctionArg> fromFunctionArgumentList(
+      int argumentsCount,
+      FunctionArg.ProtoFrom argVisitor,
+      SimpleExtension.Function declaration,
+      Function<Integer, FunctionArgument> argFunction) {
+    return IntStream.range(0, argumentsCount)
+        .mapToObj(i -> argVisitor.convert(declaration, i, argFunction.apply(i)))
+        .collect(Collectors.toList());
+  }
+
+  public ImmutableExpression.SortField fromSortField(SortField s) {
```

Comment:
minor: This should return `Expression.SortField`

Reply:
ditto